### ISSUE TITLE
"How to become a `SELECT` star"

### ISSUE THREAD

**Issue by max-sixty** [PRQL/prql#4303]
### What's up?

I was pointed towards this guide for SQL: https://wizardzines.com/zines/sql/

I'm a big fan of Julia Evans generally. I wonder whether it's possible to do something similar to this for PRQL.

Interestingly, the first page is implicitly very supportive of PRQL's approach:

<img src="https://github.com/PRQL/prql/assets/5635139/b6135008-0b19-4a4f-9384-ff417358d2db" width="300">

**Comment by snth** [comment]
Yeah, I referenced her Strange Loop keynote in my presentations last year!

![image](https://github.com/PRQL/prql/assets/652285/9b788034-0455-4f09-a7f5-a3ddd91abbf7)

**Comment by max-sixty** [comment]
Ah brilliant @snth !
